app/presenters/root_browse_page_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class RootBrowsePagePresenter
+
+  def render_for_publishing_api
+    {
+      content_id: "8413047e-570a-448b-b8cb-d288a12807dd",
+      format: "mainstream_browse_page",
+      title: "Browse",
+      locale: 'en',
+      public_updated_at: Time.now.iso8601,
```

Comment:
I'm uncomfortable with this - republishing a page shouldn't change its public_updated_at value.

I think an ideal value to use here would be the most recent time that a top-level mainstream browse page was updated.